src/server/index.ts

```diff
@@ -27,6 +29,28 @@ export default async function (options: Options): Promise<void> {
     await startWebdriverServer(config, options);
   }
 
+  if (cluster.isPrimary) {
+    if (config.storybookAutorunCmd) {
+      logger().info(
+        `Storybook should be started via \`${config.storybookAutorunCmd}\` and be accessible at ${config.storybookUrl}`,
```

Comment:
```suggestion
        `Storybook starts via \`${config.storybookAutorunCmd}\` and will be available at ${config.storybookUrl}`,
```

Reply:
It looks simpler and better. But it's important for us to show that we're waiting, because the storybook can be started manually